zig-mode.el

```diff
@@ -264,6 +264,67 @@ If given a SOURCE, execute the CMD on it."
           ;; Skip backwards past whitespace and comment end delimiters.
           (/= 0 (skip-syntax-backward " >")))))
 
+(defun zig-in-str-or-cmnt () (nth 8 (syntax-ppss)))
+(defvar zig-top-item-beg-re
```

Comment:
```suggestion
(defconst zig-top-item-beg-re
```

This doesn't have to be `defvar` as far as I can see.

Reply:
I agree, good catch

Reply:
Squashed it into the same commit.